Python/Batch/sample4_docker_swarm.py

```diff
@@ -48,6 +48,9 @@
     'resources', _STARTTASK_RESOURCE_FILE)
 _NODE_USERNAME = 'docker'
 _SSH_TUNNEL_SCRIPT = 'ssh_tunnel_batch_docker_swarm.sh'
+_TASK_USER = batchmodels.AutoUserSpecification(
```

Comment:
Might as well wrap the full UserIdentity so you don't have a hardcoded dict? (For the instances below.)